doc/api/events.md

```diff
@@ -219,7 +219,7 @@ myEmitter.emit('event');
 * `eventName` {String|Symbol} The event name
 * `listener` {Function} The event handler function
 
-The `'removeListener'` event is emitted *after* a listener is removed.
+The `'removeListener'` event is emitted *after* the listener is removed.
```

Comment:
If we are going to be more specific here, we should probably at least surround the following word "listener" in backticks to be a little more clear.